app/assets/stylesheets/spotlight/typeahead.css

```diff
@@ -37,7 +37,7 @@
 }
 
 .twitter-typeahead {
-  z-index: 1000 !important; /* get typeahead to float on top of e.g. jcrop */
```

Comment:
removed the override and things still appeared to work fine when testing on my laptop:

before:
<img width="1517" alt="screen shot 2016-12-11 at 5 18 28 pm" src="https://cloud.githubusercontent.com/assets/7741604/21085460/47c7fefa-bfc7-11e6-95bc-c56520b8ec60.png">

after removing (typeahead still falls over crop tool):
<img width="1517" alt="screen shot 2016-12-11 at 5 19 21 pm" src="https://cloud.githubusercontent.com/assets/7741604/21085465/51ac3756-bfc7-11e6-81c9-c1fcbe18a71e.png">

pushed with the line omitted entirely (and rebased on latest iiif_crop).